src/amo/pages/Guides/index.js

```diff
@@ -0,0 +1,251 @@
+              'Today’s web is tangled up with unwanted advertisements that get in your way and slow you down. Ad-blocking extensions can block or filter those ads so you can get back to distraction-free browsing.',
+            ),
+            addonCustomText: i18n.gettext(
+              'Lightweight, highly effective ad blocker. uBlock Origin enforces thousands of content filters without chewing up a bunch of memory.',
+            ),
+            exploreMore: i18n.gettext(
+              'Explore more %(linkStart)sad blockers%(linkEnd)s staff picks.',
+            ),
+            exploreUrl: '/collections/mozilla/ad-blockers/',
+          },
+          // Facebook container
+          {
+            header: i18n.gettext('Stop online trackers from stalking you'),
+            description: i18n.gettext(
+              'Online advertisers can track your activity from one website to the next, gathering information about you and your interests. Extensions can help cover your digital trail.',
+            ),
+            addonCustomText: i18n.gettext(
+              'Isolate your Facebook identity into a separate “container” to stop Facebook from tracking your activity outside of its social platform.',
+            ),
+            exploreMore: i18n.gettext(
+              'Explore more %(linkStart)sprivacy & security%(linkEnd)s staff picks.',
+            ),
+            exploreUrl: '/collections/mozilla/privacy-matters/',
+          },
+        ],
+      };
+    // The following will be addressed in another issue.
+    // See https://github.com/mozilla/addons-frontend/issues/6744.
+    case 'organize-tabs-and-bookmarks':
+    case 'enhance-your-media-experience':
+    default:
+      return {};
+  }
+};
+
+export class GuidesBase extends React.Component<InternalProps> {
+  constructor(props: InternalProps) {
+    super(props);
+
+    const { guidesSlug, guids } = this.props;
+
+    this.props.dispatch(
+      fetchGuidesAddons({
+        guids,
+        errorHandlerId: guidesSlug,
+      }),
+    );
+  }
+
+  getGuidesSectionsHtml = (
+    sections: Array<Object>,
+  ): React.ChildrenArray<React.Node> => {
+    const { addons, clientApp, i18n, lang } = this.props;
+
+    return addons.length
+      ? addons.map((addon, key) => {
+          // TODO: look into having these links use the Router (vs 'a' tag).
+          // See https://github.com/mozilla/addons-frontend/issues/6787.
+          const exploreMore = i18n.sprintf(sections[key].exploreMore, {
+            linkStart: `<a href="/${lang}/${clientApp}${
+              sections[key].exploreUrl
+            }">`,
+            linkEnd: '</a>',
+          });
+
+          return (
+            <div className="Guides-section" key={addon && addon.url}>
+              <h2 className="Guides-section-title">{sections[key].header}</h2>
+              <p className="Guides-section-description">
+                {sections[key].description}
+              </p>
+
+              <GuidesAddonCard
```

Comment:
This PR is ultra big. How about splitting it? I think you could easily omit the `GuidesAddonCard` component and ship the rest of the PR, WDYT? Maybe we could also have only one `privacy` section too?

Reply:
yep, I can definitely do that. thanks for the suggestion : )

regarding the `privacy` section suggestion - do you mean combining the `getGuids` and `getContent` function?

Reply:
no sorry, I meant just having one entry in the `sections` so that it reduces the diff even more.